packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+        "id": "48e0a2da-0999-4f2c-87f4-80432cfe6617",
+        "userName": "test.user@okta.local",
+        "externalId": "00unrudb40SOK2eup0h7",
+        "name": {
+            "givenName": "Test1",
+            "familyName": "User"
+        },
+        "displayName": "Test1 User",
+        "locale": "en_US",
+        "active": false,
+        "emails": [{
+            "primary": true,
+            "value": "test.user@okta.local",
+            "type": "work"
+        }],
+        "groups": [],
+        "meta": {
+            "created": "2019-10-18T06:50:23+00:00",
+            "lastModified": "2019-10-18T07:00:47+00:00",
+            "version": "6feac5cd8e2e3a617037e2543e3e83d0"
+        }
+    }
+
+### Deleting specific users
+
+<ApiOperation method="delete" url="/users/${userId}" />
+
+Okta does not perform a DELETE operation on `/users/${userId}`.
+
+If a user is suspended, deactivated, or removed from the application in Okta, then Okta sends a PATCH request to set the `active` attribute to `false`.
+
+## Group operations
+
+### Creating groups
+
+<ApiOperation method="post" url="/groups" />
+
+To create a group in the SCIM server, an Okta administrator must push the group using the Okta Admin Console. In the Okta Admin Console, select the SCIM application from your list of applications. In the **Push Groups** tab, click **Push Groups**. For more information, see the [Using Group Push topic](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Directory_Using_Group_Push) in the Okta Help Center.
+
+After the group is selected, Okta makes a POST request to the service provider:
+
+    POST /scim/v1/groups HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "displayName": "Test SCIM2"
+    }
+
+When it receives this request the SCIM server responds with the group details, as it would for a GET operation to the `/groups/${groupId}/`:
+
+    HTTP/1.1 201 Created
+    Date: Fri, 18 Oct 2019 07:02:44 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "74094a55-c9ee-47ae-9fd4-9137deb43497",
+        "displayName": "Test SCIM2",
+        "members": [],
+        "meta": {
+            "created": "2019-10-18T07:02:44+00:00",
+            "lastModified": "2019-10-18T07:02:44+00:00",
+            "version": "12ba4e7e158aa0b78d78c7e9ae15d917"
+        }
+    }
+
+**Note:** If the group has a description, it is sent automatically to the SCIM server:
+
+    {
+    "schemas": ["urn:scim:schemas:core:1.0", "urn:okta:custom:group:1.0"],
+        "displayName": "Group 10",
+        "urn:okta:custom:group:1.0": {
+            "description": "All Users West of The Rockies"
+        }
+    }
+
+The SCIM server is not required to save and return the group description; however there is no option to remove the description from being sent.
```

Comment:
contractions for more conversational and the removal of the semicolon - avoid the use.

The SCIM server isn't required to save and return the group description. However, there is no option to remove the description from being sent.